src/attesters/csv/csv_utils.c

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2022 Hygon Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stddef.h>
+#include <rats-tls/log.h>
+#include <openssl/evp.h>
+
+int execute_system_command(const char *cmdline)
+{
+	FILE *pipe = popen(cmdline, "r");
+	char output[4096];
+
+	if (!pipe)
+		return -1;
+
+	while (!feof(pipe))
+		fread(output, 1, sizeof(output), pipe);
+
+	pclose(pipe);
+	return 0;
+}
+
+int read_file(const char *filename, void *buffer, size_t len)
```

Comment:
The read_file is implemented in [link](https://github.com/inclavare-containers/rats-tls/blob/master/src/verifiers/sev/sev_utils.c#L12), please reuse this functiom

Reply:
I'll touch a new file src/common/utils.c that contains function read_file

Reply:
I'll insert '#include "../../verifiers/sev/sev_utils.c"' into src/attesters/csv/collect_evidence.c, and delete the definition of read_file in csv attester.  Test on these changes have been passed.